src/psyclone/tests/test_files/dynamo0p3/infrastructure/argument_mod.F90

```diff
@@ -104,6 +104,7 @@ module argument_mod
   integer, public, parameter :: GH_WRITE     = 65
   integer, public, parameter :: GH_READWRITE = 811
   integer, public, parameter :: GH_INC       = 542
+  integer, public, parameter :: GH_READINC   = 543
```

Comment:
I would suggest using the [value from the LFRic test branch](https://code.metoffice.gov.uk/trac/lfric/changeset/30105) here (420) as this will be on LFRic trunk, so the PSyclone copy is close to LFRic.

Reply:
Thanks. I didn't know what the value was so made one up.